src/main/java/com/example/expense/tracker/controller/TransactionController.java

```diff
@@ -0,0 +1,64 @@
+package com.example.expense.tracker.controller;
+
+import com.example.expense.tracker.service.TransactionService;
+import com.example.expense.tracker.model.Transaction;
+import com.example.expense.tracker.model.TransactionRequest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/home/api/transaction")
```

Comment:
`/home` can be removed. It should be  `/api/transaction`

Reply:
updated as "/api/transaction"